docs/DeveloperGuide.adoc

```diff
@@ -111,156 +111,355 @@ image::DeleteSequenceDiagram.png[]
 
 NOTE: The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 
+//@@author le0tan
 [[Design-Model]]
 === Model component
 
 .Structure of the Model Component
 image::ModelClassDiagram.png[]
 
-*API* : link:{repoURL}/src/main/java/seedu/address/model/Model.java[`Model.java`]
+*API* : link:{repoURL}/src/main/java/seedu/algobase/model/Model.java[`Model.java`]
```

Comment:
Model diagram is **comprehensive**, but perhaps it can be more **comprehensible**. Perhaps you can create 4 separate diagrams (1 for GUI, 1 for ModelManager, 1 for Plan and 1 for Problem)?

Reply:
I fixed the diagrams in [this](https://5dafe94972d6d40007016365--algobase.netlify.com/developerguide#Design-Model) preview. Can you take a look?